hugin/gdocs_updater.py

```diff
@@ -99,7 +100,7 @@ def reshape_run_info(self, runs, skiplist=[]):
                 p = [p]
             for project in p:
                 if len(project) == 0:
-                    project = 'Unknown, please check!'
+                    continue
```

Comment:
There seems to be a lag between when a card is created and when the "projects" bullet-point is populated. If this code picks up one of these cards in that state it will annoyingly add it to the checklist.
 
I don't know if there are legitimately any situations where we will have cards that never gets populated though? I guess they will be moved to "check status"